packages/seacas/libraries/ioss/src/text_mesh/Iotm_TextMesh.C

```diff
@@ -526,10 +527,10 @@ private:
     int64_t count  = 0;
     int     myProc = m_myProcessor;
 
-    for (const std::pair<int64_t, int> &elemSidePair : sideset->data) {
-      int64_t elemId = elemSidePair.first;
-      int     side   = elemSidePair.second;
-      auto    iter = std::find(m_data.elementDataVec.begin(), m_data.elementDataVec.end(), elemId);
+    for (const std::pair<EntityId, int> &elemSidePair : sideset->data) {
```

Comment:
Maybe another `const auto &elemSidePair` or even a structured binding...
`for (const auto [elemId, side] : sideset->data` (or whatever correct syntax is)